Sources/INNavigation/Navigation/Helper/View+VerticalNavigation.swift

```diff
@@ -23,15 +23,20 @@ extension View {
 		.sheet(
 			item: Binding<PresentationInfo?>(
 				get: {
-					guard binding.wrappedValue?.presentationType == .sheet else {
+					guard binding.wrappedValue?.presentationType != .fullScreen else {
```

Comment:
This will also return true when wrappedValue returns nil. I doubt that this would be correct, right? Maybe we should unwrap it first and check for all matching entries instead of inverting it? Why is this change needed at all? What types do we need to cover and why?